src/main/java/ticket/Ticket.java

```diff
@@ -33,11 +38,28 @@ public Ticket(String id, String concertHall, String eventCode, LocalDateTime tim
     }
 
     public Ticket(String concertHall, String eventCode, LocalDateTime time) {
-        setConcertHall(concertHall);
-        setEventCode(eventCode);
+        this.concertHall = concertHall;
+        this.eventCode = eventCode;
         this.time = time;
     }
 
+    public void share(String phone) {
+        System.out.println("Ticket is shared by phone: " + phone);
+    }
+
+    public void share(String phone, String email) {
+        System.out.println("Ticket is shared by phone: " + phone + " and email: " + email);
+    }
```

Comment:
move it into the end

Reply:
Fixed in https://github.com/kolosokvit/ticket-service/pull/2/commits/9f202e8e46a97a90991e72e9e9a12acca3d7ae05